cmd/publishing-bot/publisher.go

```diff
@@ -54,6 +54,11 @@ func New(cfg *config.Config) (*PublisherMunger, error) {
 		p.baseRepoPath = filepath.Join(gopath, "src", "k8s.io")
 	}
 
+	// If RULE_FILE_PATH is detected, check if the source repository include rules files.
+	if len(os.Getenv("RULE_FILE_PATH")) > 0 {
```

Comment:
@sttts with this we can load rules from the source repo... guess we need to reload when we git fetch